app/viewmodels/PostponedVatViewModel.scala

```diff
@@ -21,11 +21,17 @@ import play.api.i18n.Messages
 import services.DateTimeService
 
 object PostponedVatViewModel {
-  def apply(files: Seq[PostponedVatStatementFile])(implicit messages: Messages, dateTimeService: DateTimeService): Seq[PostponedVatStatementGroup] = {
-    val response: Seq[PostponedVatStatementGroup] = files.groupBy(_.monthAndYear).map { case (month, filesForMonth) => PostponedVatStatementGroup(month, filesForMonth) }.toList
+  def apply(files: Seq[PostponedVatStatementFile])(implicit messages: Messages,
+                                                   dateTimeService: DateTimeService): Seq[PostponedVatStatementGroup] = {
+    val response: Seq[PostponedVatStatementGroup] =
+      files.groupBy(_.monthAndYear).map {
+        case (month, filesForMonth) => PostponedVatStatementGroup(month, filesForMonth)
+      }.toList
+
     val monthList = (1 to 6).map(n => dateTimeService.systemDateTime().toLocalDate.minusMonths(n))
```

Comment:
Do we want to remove these hard coded nums? I don't think values below 10 flag as magic number warnings in scalastyle

Reply:
Done. Have created a constant named MONTHS_RANGE_ONE_TO_SIX_INCLUSIVE